### ISSUE TITLE
Create a new Next.js page called `login`

### ISSUE THREAD

**Issue by ijemmao** [nkowaokwu/igbo_api#603]
Creat a new Next.js page called `login`

**Comment by Ndohjapan** [comment]
Sure.

I can take on this.

**Comment by kingdanie** [comment]
is the page to contain placeholder content for now?

**Comment by ijemmao** [comment]
yeah, i can add more details in this ticket - if you want to pick this up @kingdanie lmk~

**Comment by kingdanie** [comment]
Yes please,

**Comment by kingdanie** [comment]
Screenshot of a draft I made:


<img width="1262" alt="Screenshot 2023-06-22 at 16 21 00" src="https://github.com/nkowaokwu/igbo_api/assets/52611355/faa26b8a-6c42-4fa3-9360-620ac6ebef45">
